packages/main/src/MenuItem.js

```diff
@@ -17,6 +17,28 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the <code>additionalText</code>, displayed in the end of the menu item.
+		 * @type {string}
+		 * @public
+		 * @since 1.8.0
+		 */
+		additionalText: {
+			type: String,
+		},
+
+		/**
+		 * Defines whether an additional text should be hidden.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 * @since 1.8.0
+		 */
+		 hideAdditionalText: {
```

Comment:
Is this expected by default to be shown? Because with default false on hide, this means that by default we will have it shown?

Reply:
yes, that is the default behavior - you have to set it explicitly to true to hide the text (for example - on mobile to hide shortcuts because there is no physical kbd)